scripts/generate_schemas.py

```diff
@@ -288,22 +334,25 @@ def run(self):
         with tarfile.open(self.sourceTar, "r") as tarball:
             tarball.extractall(self.tmpDir)
         d = os.path.join(self.schemaDir, "src/main/resources/avro")
-        # TODO should we make it an option to download this each time
-        # or to use a local copy? Should we continue using this version?
-        url = "http://www.carfab.com/apachesoftware/avro/stable/java/"\
-            "avro-tools-1.7.7.jar"
-        # it's useful to put a copy in cwd and uncomment this for
-        # developement; should we put this in as an official option?
-        # self.avroJar = os.path.join(os.getcwd(), "avro-tools.jar")
-        self.download(url, self.avroJar)
+        if self.avroJarPath:
```

Comment:
I think we should prefer the more explicit `self.avroJarPath is not None` here. This reads to me like we expect the variable is a boolean, whereas the comparison with None is much more obvious. See also [pep8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations).